evcouplings/compare/hmmer_wrappers.py

```diff
@@ -0,0 +1,309 @@
+from evcouplings.align import read_hmmer_domtbl
+import pandas as pd
+from collections import namedtuple
+from evcouplings.utils.system import (
+    run, create_prefix_folders, verify_resources, temp
+)
+
+from os import path
+
+
+# output fields for storing results of a jackhmmer run
+# (returned by run_jackhmmer)
+HmmbuildResult = namedtuple(
+    "HmmbuildResult",
+    ["prefix", "hmmfile", "output"]
+)
+
+
+def run_hmmbuild(msafile, prefix,
+                  cpu=None,
```

Comment:
Indentation wrong, please also check if there are any more relevant hmmbuild parameters that we might want to wrap with the function